Control coverage/Feature/VirtualMachine.md

```diff
@@ -17,3 +16,9 @@
+| <b>ControlId:</b><br>Azure_VirtualMachine_SI_Deploy_GuestConfig_Extension<br><b>DisplayName:</b><br>Guest Configuration extension must be deployed to the VM using Azure Policy assignment. <br><b>Description: </b><br> Guest Configuration extension must be deployed to the VM using Azure Policy assignment. | <b> ARM API to list Virtual Machines at subscription level:</b> <br> /subscriptions/{subscriptionId}/providers/Microsoft.Compute<br> /virtualMachines?api-version=2019-07-01<br><b>Properties:</b><br> identity.type <br> <br> <b> ARM API to list Virtual Machine Extensions at resource level: </b> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}<br>/providers/Microsoft.Compute/virtualMachines/{vmName}/extensions? <br> api-version=2019-07-01 <br><b>Properties:</b><br> properties.type <br> properties.publisher | <b>Passed: </b><br> Guest config extension is present, and <br> System assigned MI is enabled for VM.<br><b>Failed: </b><br> Guest config extension is not present in VM, or <br> System assigned MI is disabled for VM.<br><b>NotApplicable: </b><br>VM is part of ADB cluster and AKS VMs. |
+| <b>ControlId:</b> <br>Azure_VirtualMachine_SI_Remediate_Assessment_Soln_Vulnerabilities <br> <b>DisplayName: </b> <br>Vulnerabilities must be remediated by a Vulnerability Assessment solution. <br><b>Description: </b><br> Vulnerabilities must be remediated by a Vulnerability Assessment solution. | <b>ARM API to list security assessments at subscription<br> level:</b><br>/subscriptions/{subscriptionId}/providers/Microsoft.Security<br>/assessments?api-version=2020-01-01<br><b>Properties:</b><br>id, name, resourceDetails/Id, displayName, status/code,<br> status, additionalData | <b>Scope: </b> Applies to all Virtual Machine resources.<br><br><b>Config: </b> AssessmentName: 71992a2a-d168-42e0-b10e-6b45fa2ecddb <br><br> <b>Passed: </b><br>ASC assessment found with Healthy status code.<br><br> <b>Failed: </b><br>ASC assessment found with Unhealthy status code. |
```

Comment:
Lets use '.' to access child properties.

Reply:
Done.